source/reference/minio-mc/mc-batch-generate.rst

```diff
@@ -74,8 +74,11 @@ Parameters
    
    The type of job to generate a YAML document for.
    
-   Currently, :mc:`mc batch` supports the ``replicate`` and ``keyrotate`` job types.
-
+   Supports the following values:
+   
+   -  ``replicate``
```

Comment:
Should these be links?
Either to the sections below or the administration/batch-replicate docs?

I think surfacing links to the description in administration somewhere other than the bottom of each YAML is worth doing.

Reply:
:thinking: yeah I can add some reflinks to make this work.